Library/Formula/dfix.rb

```diff
@@ -0,0 +1,21 @@
+class Dfix < Formula
+  desc "An auto-upgrade tool for D source code"
+  homepage "https://github.com/Hackerpilot/dfix"
+  url "https://github.com/Hackerpilot/dfix.git",
+      :tag => "v0.2.2",
+      :revision => "ee55825b695fce94a337ab6a2009366372649d46"
+
+  head "https://github.com/Hackerpilot/dfix.git", :shallow => false
+
+  depends_on "dmd" => :build
+
+  def install
+    system "make"
+    system "make", "test"
+    bin.install "bin/dfix"
+  end
+
+  test do
+    system "dfix", "--help"
```

Comment:
Also, as discussed earlier, please update this test to operate on an actual file you write out.